Microsoft.Toolkit.Uwp.UI/Extensions/Visual/VisualExtensions.cs

```diff
@@ -0,0 +1,738 @@
+
+        private static void OnAnchorPointChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            if (e.NewValue is string str)
+            {
+                SetAnchorPoint(d, str);
+            }
+        }
+
+        private static void OnCenterPointChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            if (e.NewValue is string str)
+            {
+                SetCenterPoint(d, str);
+            }
+        }
+
+        private static void OnOffsetChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            if (e.NewValue is string str)
+            {
+                SetOffset(d, str);
+            }
+        }
+
+        private static void OnOpacityChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            if (e.NewValue is double dbl)
+            {
+                SetOpacity(d, dbl);
+            }
+        }
+
+        private static void OnRotationAngleChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            if (e.NewValue is double dbl)
+            {
+                SetRotationAngle(d, dbl);
+            }
+        }
+
+        private static void OnRotationAngleInDegreesChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            if (e.NewValue is double dbl)
+            {
+                SetRotationAngleInDegrees(d, dbl);
+            }
+        }
+
+        private static void OnRotationAxisChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            if (e.NewValue is string str)
+            {
+                SetRotationAxis(d, str);
+            }
+        }
+
+        private static void OnScaleChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            if (e.NewValue is string str)
+            {
+                SetScale(d, str);
+            }
+        }
+
+        private static void OnSizeChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            if (e.NewValue is string str)
+            {
+                SetSize(d, str);
+            }
+        }
+
+        private static void OnNormalizedCenterPointChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            if (d is FrameworkElement element && !DesignTimeHelpers.IsRunningInLegacyDesignerMode)
+            {
+                SetupNormalizedCenterPoint(e, element);
+            }
+        }
+
+        private static void SetupNormalizedCenterPoint(DependencyPropertyChangedEventArgs e, FrameworkElement element)
+        {
+            element.SizeChanged -= KeepCenteredElementSizeChanged;
+
+            if (e.NewValue is string normalizedValue)
+            {
+                element.SizeChanged -= KeepCenteredElementSizeChanged;
```

Comment:
you have already done this on line 606. Am I missing something?

Reply:
Good catch, fixed